storage/new_bucket/main.tf

```diff
@@ -32,14 +32,12 @@ resource "google_storage_bucket" "static" {
 # [END storage_create_new_bucket_tf]
 
 # [START storage_upload_object_tf]
-# Upload files
-# Discussion about using tf to upload a large number of objects
-# https://stackoverflow.com/questions/68455132/terraform-copy-multiple-files-to-bucket-at-the-same-time-bucket-creation
+# Creates a text file based on content_type with content
```

Comment:
I would change this to:

```
# Create a new object in a storage bucket
```

to be more consistent with the heading of the doc ("Upload an object to a bucket").

Reply:
The sample comments don't need to match the headings in the document. The sample does a specific thing, the documentation should be talking about the general with the explicit specific thing being done here and how it can be customized.

Reply:
I disagree: I think this sample should at least be somewhat consistent with the headings in the document. If this sample were used elsewhere other than Cloud Storage docs, my opinion would change. But from a usability/user experience perspective, having a consistently named comment in a sample makes the entire page more cohesive. Just my two cents, as I don't own these samples anymore.